docs/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart
+
+## Run bblfsh and gitbase Dependencies
+
+It is recommended to read about `bblfsh` and `gitbase` from its own documentation, but here is a small guide about how to run both easily:
+
+Launch [bblfshd](https://github.com/bblfsh/bblfshd) and install the drivers. More info in the [bblfshd documentation](https://doc.bblf.sh/user/getting-started.html):
+
+```bash
+$ BBLFSHD_CONTAINER=bblfsh
+$ BBLFSHD_IMAGE=bblfsh/bblfshd;
+$ docker run --privileged
+    --publish 9432:9432
+    --volume /var/lib/bblfshd:/var/lib/bblfshd
+    --name ${BBLFSHD_CONTAINER}
+    ${BBLFSHD_IMAGE};
```

Comment:
why some commands end with `;` and others not?

Reply:
because you looked deeper than I did ;)
Fixed; thanks!!!